internal/transactions/send.go

```diff
@@ -95,8 +95,14 @@ func send(
 
 	signerName := sendFlags.Signer
 
-	if signerName == "" && proposer == nil && payer == nil && len(authorizers) == 0 {
-		signerName = state.Config().Emulators.Default().ServiceAccount
+	if signerName == "" {
+		if proposer == nil && payer == nil && len(authorizers) == 0 {
+			signerName = state.Config().Emulators.Default().ServiceAccount
```

Comment:
Will this not be wrong and give a bad error message if network is not emulator?

Reply:
it is current behaviour, we can change it ofc but probably people rely on that.

Reply:
True. The diff fooled me.